src/msg/async/AsyncConnection.cc

```diff
@@ -435,15 +435,9 @@ void AsyncConnection::process()
           recv_stamp = ceph_clock_now();
           ldout(async_msgr->cct, 20) << __func__ << " begin MSG" << dendl;
           ceph_msg_header header;
-          ceph_msg_header_old oldheader;
           __u32 header_crc = 0;
-          unsigned len;
-          if (has_feature(CEPH_FEATURE_NOSRCADDR))
```

Comment:
@yuyuyu101 could you help remove the code handling `CEPH_FEATURE_NOSRCADDR` in `Pipe.cc` as well?

Reply:
done